frontend/lib/src/components/widgets/DataFrame/columns/utils.test.ts

```diff
@@ -245,8 +245,13 @@ describe("formatNumber", () => {
     [10.1234, "10.1234"],
     // Rounds to 4 decimals
     [10.12346, "10.1235"],
+    [0.00016, "0.0002"],
+    // If number is smaller than 0.0001, shows the next decimal number
+    // to avoid showing 0 for small numbers.
+    [0.0000051, "0.000005"],
+    [0.000000123, "0.0000001"],
```

Comment:
Should we include a rounding up example here?

Reply:
I added more examples